examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -0,0 +1,445 @@
+      "text/plain": [
+       "<fresnel.util.ImageArray at 0x12aff8910>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import fresnel\n",
+    "\n",
+    "# Color particles by different local environments identified\n",
+    "colors = np.empty((snapshot.particles.N, env_cluster.num_clusters))\n",
```

Comment:
This happens to work because there are three color channels (RGB) and three clusters. However, the shape shouldn't vary with the number of clusters and should always be `(N, 3)`.
```suggestion
    "colors = np.empty((snapshot.particles.N, 3))\n",
```